spacy_experimental/multiembed/multiembed.py

```diff
@@ -0,0 +1,206 @@
+import thinc
+
+from spacy.tokens import Doc
+from thinc.api import Model, chain, with_array
+from thinc.api import list2ragged, ragged2list, concatenate
+from thinc.types import Floats2d, DTypes, Ints2d, Array2d
+from typing import Optional, Callable, List, Dict, Any
+from typing import Union, Sequence, Tuple
+
+Embed = thinc.registry.layers.get("Embed.v1")
+Maxout = thinc.registry.layers.get("Maxout.v1")
+Dropout = thinc.registry.layers.get("Dropout.v1")
+Vectors = thinc.registry.layers.get("spacy.StaticVectors.v2")
+Extract = thinc.registry.layers.get("spacy.FeatureExtractor.v1")
+
+
+InT = Union[Sequence[Any], Array2d]
+OutT = Ints2d
+
+
+@thinc.registry.layers("remap_ids.v2")
```

Comment:
Hm, it feels strange to add to the Thinc registry from within `spacy-experimental`. I suppose we need this for config files though. Then maybe it should be `spacy_experimental.remap_ids` for now?

Reply:
This whole PR should just wait until the next thinc release.

Reply:
Thinc 8.1.1 is now available, Ákos :-)